### DISCUSSION TITLE
[Goland/VS Code] "Copilot is encountering temporary issues"

### DISCUSSION THREAD

**Discussion by mateusmarquezini** [community/community#17693] [Copilot Conversations] [answered]
Hey guys, 

Maybe you may saw this error many times over here. Today I was trying GitHub Copilot for the first time on Goland and I couldn't use it, I was getting the following error: _Copilot is encountering temporary issues_

I don't know what's going on, whether this is a temporary issue or something related to my account. I don't know, 2 days ago Copilot was working fine on VS Code, but today it stopped working, even on VS Code.

I have access to the technical preview normally.

Did someone here already get this error and was able to solve it?

I appreciate any help! Thx!

**Comment by davecheney** [comment, marked as answer]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by mateusmarquezini** [reply]
humm, maybe @davecheney, I'm not sure, I'm currently using VPN, which might be the problem. I'll try later on my personal notebook and I back here with feedback. Thanks for now.

**Reply by mateusmarquezini** [reply]
seems that was the reason, I need to figure out a workaround for that. Thanks!
